.github/workflows/push-tags.yaml

```diff
@@ -0,0 +1,66 @@
+name: Push Tag Build
```

Comment:
I'm thinking we want this action to only tag, and do nothing else. It would keep this new action simple, and I like that the existing action in push-build.yaml owns publishing the build, and that it would be triggered by the tag. Or was there a need for this action to both tag and build?

Reply:
okay, now this action only trigger the action to create tag